docs/source/index.rst

```diff
@@ -6,7 +6,9 @@ Podman_ is a daemonless, open source, Linux native tool designed to make it easy
 
 Containers under the control of Podman can either be run by root or by a non-privileged user. Podman manages the entire container ecosystem which includes pods, containers, container images, and container volumes using the libpod_ library. Podman specializes in all of the commands and functions that help you to maintain and modify OCI container images, such as pulling and tagging. It allows you to create, run, and maintain those containers and container images in a production environment.
 
-The Podman service runs only on Linux platforms, however a REST API and clients are currently under development which will allow Mac and Windows platforms to call the service. There is currently a RESTful based remote client which runs on Mac or Windows platforms that allows the remote client to talk to the Podman server on a Linux platform. In addition to those clients, there is also a Mac client.
+There is a RESTFul API to manage containers.  We also have a remote Podman client that can interact with
+the RESTFul service.  We currently support clients on Linux, Mac, and Windows.  The RESTFul service is only
```

Comment:
I think we need to clarify this more - this makes it sound like Podman (that is not the REST API) can run on non-Linux platforms.

Reply:
@baude pointed me here from #9104, and while this is far better than the wording I was concerned about in my issue,  I agree that this last sentence about the RESTFul service only being available on Linux sounds a little ambiguous.